readinglist/backend/__init__.py

```diff
@@ -36,6 +36,16 @@ def get_all(self, resource, user_id, filters=None, sorting=None,
                 pagination_rules=None, limit=None):
         raise NotImplementedError
 
+    def strip_deleted_record(self, resource, user_id, record):
+        """Strip the record to its deleted version (in place)."""
```

Comment:
I don't get what this means from the comment.

It seems to remove all the fields of the record which aren't the id or the modification field. Is that correct?